vagrant_resources/vagrant/Vagrantfile.virtualbox

```diff
@@ -28,15 +28,29 @@ Vagrant.configure("2") do |config|
     auto_correct: true
   config.vm.network :private_network,
     ip: "192.168.33.10"
-  if VM_SIZE == "512mb"
-    config.vm.box = "tl-debian-jessie-64bit-20gb"
-    config.vm.box_url = "https://s3-us-west-1.amazonaws.com/terminal-labs-vagrant-boxes/tl-debian-jessie-64bit-20gb.box"
-  elsif VM_SIZE == "1024mb"
-    config.vm.box = "tl-debian-jessie-64bit-30gb"
-    config.vm.box_url = "https://s3-us-west-1.amazonaws.com/terminal-labs-vagrant-boxes/tl-debian-jessie-64bit-30gb.box"
-  elsif VM_SIZE == "2048mb"
-    config.vm.box = "tl-debian-jessie-64bit-40gb"
-    config.vm.box_url = "https://s3-us-west-1.amazonaws.com/terminal-labs-vagrant-boxes/tl-debian-jessie-64bit-40gb.box"
+  if VM_HOST == "debian-8"
```

Comment:
This works, but is big and will blow up with more OSs. I'll make another ticket for condensing it.